src/vtcomposite.cpp

```diff
@@ -614,21 +650,35 @@ struct InternationalizeWorker : Napi::AsyncWorker
 
             while (auto layer = tile.next_layer())
             {
-                // TODO short circuit if hidden attributes not present?
+                // TODO short circuit if hidden attributes not present? (call vtzero's add_existing_layer)
                 vtzero::layer_builder lbuilder{tbuilder, layer.name(), layer.version(), layer.extent()};
                 while (auto feature = layer.next_feature())
                 {
                     vtzero::geometry_feature_builder fbuilder{lbuilder};
-
                     fbuilder.copy_id(feature);
                     fbuilder.set_geometry(feature.geometry());
 
+                    bool keep_feature = true;
                     bool name_was_set = false;
                     vtzero::property_value name_value;
                     while (auto property = feature.next_property())
                     {
                         std::string property_key = property.key().to_string();
 
+                        if (property_key == "_mbx_worldview")
+                        {
+                            if (property.value().type() == vtzero::property_value_type::string_value)
+                            {
+                                keep_feature = processWorldview(static_cast<std::string>(property.value().string_value()), fbuilder);
+                            }
+                            else
+                            {
+                                keep_feature = false;
```

Comment:
Right, the variable is still needed since the feature commit happens after the loop exits so just an addition of break will suffice in the else block after toggling the keep_feature variable.